src/Reveal.Sdk.Dom.Tests/Data/DataSourceItems/AmazonAthenaDataSourceItemFixture.cs

```diff
@@ -0,0 +1,97 @@
+using Reveal.Sdk.Dom.Data;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSourceItems
+{
+    public class AmazonAthenaDataSourceItemFixture
+    {
+        [Fact]
+        public void Constructor_SetsTitleAndDataSource_ValidDataSource()
+        {
+            // Arrange
+            string title = "Test Item";
+            var dataSource = new AmazonAthenaDataSource();
+            var item = new AmazonAthenaDataSourceItem(title, dataSource);
+
+            // Act
+            var actualTitle = item.Title;
+            var actualDataSource = item.DataSource;
+
+            // Assert
+            Assert.Equal(title, actualTitle);
+            Assert.Equal(dataSource, actualDataSource);
+        }
+
+        [Fact]
+        public void Constructor_SetsTitleAndDataSource_NullDataSource()
```

Comment:
As discussed above, I think just remove the Null test cases, as this class only uses the parent class's constructor and doesn't do anything more.